### ISSUE TITLE
improve SoarUnit failure output

### ISSUE THREAD

**Issue by garfieldnate** [soartech/jsoar#103]
SoarUnit outputs something like this for a failed test:

    FAILED: test xyz returns nil with total 0, never called (pass) function. Ran 101 decisions. (0.378 s)

The main thing missing is the line number that the test occurs on. I'm trying to write a text editor plugin for running SoarUnit, and having the line number would allow the user to click and go straight to the problem test.

Also, I don't understand what "returns nil with total 0" means in this context. This output is not documented as far as I can see.

**Comment by marinier** [comment]
Agreed, the text output needs work. I suspect it's gone unnoticed for so
long because I'm the primary soarunit user, and I always use the GUI.

As it is, you can probably use the test name to find the test (although
note that there's nothing that prevents multiple tests from having the same
name).
